docs/references.bib

```diff
@@ -1,3 +1,4 @@
+% Encoding: UTF-8
 
```

Comment:
```suggestion

```

Reply:
Thanks a lot: I hope this does the trick.